.meta/sinks/gcp_stackdriver_logs.toml.erb

```diff
@@ -113,6 +113,25 @@ Exactly one of `billing_account_id`, `folder_id`, `organization_id`, or \
 `project_id` must be set.\
 """
 
+[sinks.gcp_stackdriver_logs.options.severity_key]
+type = "string"
+required = false
+examples = ["severity"]
+description = """\
+The field of the log event from which to take the outgoing log's \
+`severity` field. The named field is removed from the log event if \
+present, and must be either an integer between 0 and 800 or a string \
+containing one of the severity level names (case is ignored) \
```

Comment:
Can you link to the severity levels here?

Reply:
I believe I did just below that, but I'll add another link.